bin/storm

```diff
@@ -312,6 +312,13 @@ def repl():
     cppaths = [CLUSTER_CONF_DIR]
     exec_storm_class("clojure.main", jvmtype="-client", extrajars=cppaths)
 
+def get_logback_conf_dir():
+    cppaths = [CLUSTER_CONF_DIR]
+    storm_logback_conf_dir = confvalue("storm.logback.conf.dir", cppaths)
+    if(storm_logback_conf_dir == None or storm_logback_conf_dir == "nil"):
```

Comment:
Here storm_logback_conf_dir == "nil" will match if the storm_logback_conf_dir is set to  "nil" . I think you meant to check if the string is empty, a better way probably is "if not storm_logback_conf"